README.md

```diff
@@ -1,2 +1,24 @@
 # hw01
-MY name is David 
\ No newline at end of file
+#<Biography of David Kanos>
+I was born on *April 24th, 1996* in **Jos, Nigeria** 
+I am currently getting my Masters in Public Policy at **The University of Chicago**
+
+##<Family>
+I have two younger sisteres named *Elsa* and *Natalia* and I love them both very much.
+My mum runs and NGO in Nigeria that helps widows and orphans affected by Boko Haram in Nigeria.
+My dad is an investor and consoultant in Nigeria. 
+
+###<My Hobbies>
+*Working out 
+*Cooking 
+*Reading
+*I play 8 sports
+*Hiking
+*Traveling
+
+####<My COVID summer>
+This summer I worked as an intern for the Brookings Institution, and the United Nations Development Program for Africa where I conducted research on the Effect of COVID-19 on Africa. It was an extreemly fulfiling internship, and below is the link to some of my work at Brookings. 
+https://harris.uchicago.edu/admissions/blog/my-summer-internship-brookings-institute - automatic!
+[Brookings](https://harris.uchicago.edu/admissions/blog/my-summer-internship-brookings-institute)
+
+This excersise was not too cumbersome once I understood what I was doing (Thanks to Debi) I thought that once the final document was produced it would look a bit different/more aestetically pleaseing but I guess not. Hopefully I did the assignment properly as I followed the instructions from the cheat sheet. Alltogether I thought this was a good excersise, and I think it is a lot easier for me to produce my code locally rather than via the server.
```

Comment:
Hi David,

This all looks good to me! If you could add a picture to let more people know about your face. It would be awesome!

Nice Job!

Yuwen